index.html

```diff
@@ -0,0 +1,720 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/793d8924a3.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap">
+    <title>TNW</title>
+</head>
+
+<body>
+    <nav class="flex">
+        <ul class="flex">
+            <li class="color-y">News</li>
+            <li class="color-y">Events </li>
+            <li class="color-y">Business </li>
+            <li class="color-y">AMAs</li>
+            <li class="color-y">Spaces</li>
+            <li class="color-y">Terms & Conditions</li>
+        </ul>
+        <ul class="flex top-right">
+            <li class="color-y">About</li>
+            <li class="color-y">Advertise</li>
+            <li class="color-y">Jobs</li>
+            <li class="color-y">Contact</li>
+        </ul>
+    </nav>
+    <div class="flex nav-2">
+        <div class="nav-2-left-holder">
+            <img class="hidden-1 nav-img" id="overwrite" alt="The Next Web"
+                src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+            <ul class="flex">
+                <li class="color-y">LATEST</li>
+                <li class="color-y">HARD FORK</li>
+                <li class="color-y">PLUGGED</li>
+                <li class="color-y">README</li>
+                <li class="color-y">GROWTH QUARTERS</li>
+                <li class="color-y">NEURAL</li>
+            </ul>
+        </div>
+        <div class="flex nav-2-right">
+            <i class="fas fa-search fa-flip-horizontal nav-2-icon"></i>
+            <i class="fas fa-bars nav-2-icon-2 hidden-2" id="hidden-2"></i>
+        </div>
+    </div>
+
+    <div class="top-grid">
+        <div class="top-grid-1">
+            <div class="child-sec">
+                <div class="head-text">
+                    <div class="logo-date">
+                        <div class="tmw-logo"><img src="./images/tnw.svg" alt="logo"></div>
+                    </div>
+                    <p class="current-date white-color">Wednesday - 5, August 2020</p>
+                    <div class="move-content">
+                        <div class="headings">INSIGHTS</div>
+                        <div class="head-text-title">Kodak, now a pharma company, expects drugs to make up for 30-40% of
+                            its
+                            future business.</div>
+                        <div class="head-text-bottom">by Ivan Mehta <i class="far fa-clock"></i> 1 hour ago <i
+                                class="fas fa-share-alt"></i> 6</div>
+                    </div>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] The logo should appear clearly in all pixels bigger than thee tablet view.

| Yours | Original |
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/61479648/89557266-edc95280-d7e8-11ea-8c45-bb959216cbcd.png) | ![image](https://user-images.githubusercontent.com/61479648/89557297-f588f700-d7e8-11ea-8917-95f8f1bee285.png) |

- [ ] Also fix the gradient.

![image](https://user-images.githubusercontent.com/61479648/89557360-10f40200-d7e9-11ea-832b-3aaa7d36e7a0.png)